pkgs/development/libraries/liblinphone/use-normal-jsoncpp.patch

```diff
@@ -0,0 +1,57 @@
+From ea6901c9ed0f804bd2d6d09e514610518c2f4f09 Mon Sep 17 00:00:00 2001
+From: Lorenz Brun <lorenz@brun.one>
+Date: Fri, 18 Mar 2022 17:36:22 +0100
+Subject: [PATCH] Use normal jsoncpp
+
+---
+ src/CMakeLists.txt    | 6 +++---
+ tester/CMakeLists.txt | 2 +-
+ 2 files changed, 4 insertions(+), 4 deletions(-)
+
+diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
+index 5cd87fe9a..822b2151e 100644
+--- a/src/CMakeLists.txt
++++ b/src/CMakeLists.txt
+@@ -34,7 +34,7 @@ if (ENABLE_FLEXIAPI)
+ 	if (XCODE)
+ 		list(APPEND LINK_LIBS jsoncpp_static)
+ 	else()
+-		list(APPEND LINK_LIBS jsoncpp_object)
++		list(APPEND LINK_LIBS jsoncpp)
```

Comment:
The other alternative is re-enabling `BUILD_OBJECT_LIBS` in `jsoncpp`: https://github.com/austinbutler/nixpkgs/blob/8726b4987c4160f9f039ac475a88cad8d0eba1af/pkgs/development/libraries/jsoncpp/default.nix#L38

It was turned off in https://github.com/NixOS/nixpkgs/pull/102379, but I wonder if it's worth having compatibility issues that require patches like this.

Reply:
I know, I initially flipped that. But what this does is statically link object files from another package which is not really wanted here.